pontoon/base/views.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime
```

Comment:
Yeah, the other convention also tells to put things from stdlib above all project dependencies. Will do.